From 577a266dee5a5b5c33c41695376e1e009eadb505 Mon Sep 17 00:00:00 2001
From: Zhou Jingyu <b02241@freescale.com>
Date: Wed, 6 May 2009 13:59:38 +0800
Subject: [PATCH] ENGR00112215 Add support for MC13892 2.0 board in 26 kernel

Add support for MC13892 2.0 board in 26 kernel.
In MC13892 2.0 board, VUSB2 should keep active for 2.6V.
In MC13892 1.0 board, VIOHI is used instead

Signed-off-by: Zhou Jingyu <Jingyu.Zhou@freescale.com>
---
 arch/arm/mach-mx51/mx51_3stack.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx51/mx51_3stack.c b/arch/arm/mach-mx51/mx51_3stack.c
index 3eb2df1..c0f6a92 100644
--- a/arch/arm/mach-mx51/mx51_3stack.c
+++ b/arch/arm/mach-mx51/mx51_3stack.c
@@ -152,6 +152,7 @@ static int __init mc13892_reg_int(void)
 		if ((strcmp(reg_name[i], "VIOHI") == 0) ||
 			(strcmp(reg_name[i], "VPLL") == 0) ||
 			(strcmp(reg_name[i], "VDIG") == 0) ||
+			(strcmp(reg_name[i], "VUSB2") == 0) ||
 			(strcmp(reg_name[i], "VGEN2") == 0))
 			continue;
 		regulator = regulator_get(NULL, reg_name[i]);
-- 
1.5.4.4

